normandy/recipes/models.py

```diff
@@ -293,10 +294,15 @@ def locale(self):
 
     @cached_property
     def country(self):
-        try:
-            ip_address = self.request.META['HTTP_X_FORWARDED_FOR'].split(',')[0]
-        except (KeyError, IndexError):
+        if settings.NUM_PROXIES == 0:
```

Comment:
This might fit better in a utility method, like `get_client_ip(request)` or something.

It'd be easier to test too.